opentelemetry-otlp/tests/integration_test/tests/metrics.rs

```diff
@@ -293,7 +293,7 @@ mod tests {
         // Set up the exporter
         let exporter = create_exporter();
         let reader = PeriodicReader::builder(exporter)
-            .with_interval(Duration::from_millis(100))
+            .with_interval(Duration::from_secs(30))
```

Comment:
putting a large interval here, to make sure the test is actually testing shutdown triggered exporting, not a timer triggered.

Reply:
Good idea